### ISSUE TITLE
Investigate/Confirm Fields on Claims API (Claims Step Expansion)

### ISSUE THREAD

**Issue by StacyB2023** [department-of-veterans-affairs/va-mobile-app#8796]
## Description 

**Overview:**

-  VA.gov is moving the current state of the claims functionality from a **5 Step Claims Process to an 8 Step Process**

- In order to have a seamless user journey, we would like the mobile application to reflect the same (where possible) to VA.gov. As a first step, the following data and information needs to be validated in the claims API

**Tasks/Acceptance Criteria**

1. Verify that the following fields are available in the **overview index endpoint** for Claims_

**- "phase"**

**- “phaseChangeDate”**

**- “documentsNeeded”**

**- “developmentLetterSent”**

2. Supporting Tickets: If the fields listed above are missing from the overview index endpoint, note additional effort/steps/tickets needed to ADD these into the the BE **list view** for Claims API (**list response**)

**Comment by aherzberg** [comment]
The index endpoint has all fields with the exception of the phase field. Lighthouse is already working on adding the `latestPhaseType` to the index endpoint (according to Jerek from the benefits team they are working on it this sprint). created new ticket to implement once that is done. https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8822
